docs/changelog.md

```diff
@@ -0,0 +1,31 @@
+# Changelog
+
+## Breaking Changes
+
+PydanticAI is still pre-version 1, so breaking changes will occur, however:
+
+- We try to minimize them as much as possible.
+- We will use minor version bumps to signify breaking changes.
+- Wherever possible we'll deprecate old features so code continues to work with deprecation warnings when changing the public API.
+- We intend to release V1 in summer 2025, and then follow strict semantic versioning, e.g. no intentional breaking changes except in major versions.
+
+### V0.1.0
+
+See [#1248](https://github.com/pydantic/pydantic-ai/pull/1248) — the attribute/parameter name `result` was renamed to `output` in many places. Hopefully all changes keep a deprecated attribute or parameter with the old name, so you should get many deprecation warnings.
+
+See [#1484](https://github.com/pydantic/pydantic-ai/pull/1484) — `format_as_xml` was moved and made available to import from the package root, e.g. `from pydantic_ai import format_as_xml`.
```

Comment:
should these be in the opposite order? I'd think newer first is better than newer last?

Reply:
I think we want in order of importance.

Reply:
I guess as long as it's ordered by version, and then just importance within a single version, that's fine.